src/plugins/intel_gpu/src/graph/data.cpp

```diff
@@ -92,7 +92,7 @@ void data_inst::load(BinaryInputBuffer& ib) {
         pos += data_size;
         ib.seekg(pos);
     } else {
-        _outputs[0] = get_network().get_memory_pool().get_memory(output_layout, _allocation_type, false);
+        _outputs[0] = get_network().get_engine().allocate_memory(output_layout, _allocation_type, false);
```

Comment:
Is this change has any impact?

Reply:
Actually, this change has no impact on performance and any functional behavior.
The purpose of this change is to allocate constant data memory with the same procedure as when not using the cache.